helm/keycloak/values/values-quickstart.yaml

```
@@ -42,9 +42,9 @@ probeInitialDelaySeconds: 360
 disableSsl: true
 
 resources:
-   requests:
-     cpu: 100m
-     memory: 128Mi
+  requests:
+     cpu: 1000m
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)